govstack_api/graphql_api_client.py

```diff
@@ -0,0 +1,34 @@
+from types import SimpleNamespace
+from unittest import mock
+from graphene.test import Client
+from graphene import Schema
+
+
+class GrapheneClient:
+    def __init__(self, request, query, mutation):
+        self._client = Client(Schema(query=query, mutation=mutation))
+        self.context = self.get_context(request)
+
+    def create_base_context(self):
+        user = mock.Mock(is_anonymous=False)
+        user.has_perm = mock.MagicMock(return_value=False)
```

Comment:
why is this part needed? User is always replaced with request.user, I think it an be removed

Reply:
You are right. I will delete "def create_base_context()" and replace it with `context = SimpleNamespace(user=request.user)`

Reply:
I think we can simplify it to:
```
    def get_context(self, request):
        return SimpleNamespace(user=request.user)
```